cpp/src/column/column_factories.cpp

```diff
@@ -52,6 +52,25 @@ std::unique_ptr<column> make_numeric_column(
       std::vector<std::unique_ptr<column>>{});
 }
 
+// Create column with existing null mask
+std::unique_ptr<column> make_numeric_column(
+    data_type type, size_type size,
+    bitmask_type const* null_mask, size_type null_count,
+    cudaStream_t stream, rmm::mr::device_memory_resource* mr) {
+  CUDF_EXPECTS(is_numeric(type), "Invalid, non-numeric type.");
+
+  // copy null mask
+  rmm::device_buffer copy_null_mask;
+  if( null_mask != nullptr )
+      copy_null_mask = rmm::device_buffer( null_mask,
+                                           bitmask_allocation_size_bytes(size),
+                                           stream, mr);
+  return std::make_unique<column>(
+      type, size, rmm::device_buffer{size * cudf::size_of(type), stream, mr},
+      copy_null_mask, null_count,
+      std::vector<std::unique_ptr<column>>{});
```

Comment:
This is already defaulted in the `column` ctor, so no need to have this. 
```suggestion
      );
```